notebooks/0-the-cloud/0-the-cloud.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>I think we'll want to keep the answers in a separate Notebook. Otherwise it's too easy for people to "look ahead" and not learn much</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>yeah putting the answers else where was always my plan, I just put them here so we knew what was expected</p>